### DISCUSSION TITLE
RSBot v2.6

### DISCUSSION THREAD

**Discussion by myildirimofficial** [myildirimofficial/RSBot#567] [Announcements]
## What's Changed
* Some changes for the pending queque by @Munderstand101 in https://github.com/SDClowen/RSBot/pull/536
* Add option to auto select weaker mobs first in case of emergency by @ngoedde in https://github.com/SDClowen/RSBot/pull/537
![image](https://user-images.githubusercontent.com/12081190/199255240-bf3b1601-394a-48ab-81a5-b578c637c61f.png)
* Add lookup table for collisions by @ngoedde in https://github.com/SDClowen/RSBot/pull/539
* Worked on merged pr536, Also fixed dc while pending, fix translations, fix pending timer by @SDClowen in https://github.com/SDClowen/RSBot/pull/540
* Fix null reference exception while bot starting by @SDClowen in https://github.com/SDClowen/RSBot/pull/541
* Fix & Optimized auto join to party matching by condition. by @SDClowen in https://github.com/SDClowen/RSBot/pull/542
* Fix command still busy by @ngoedde in https://github.com/SDClowen/RSBot/pull/543
* Added feature, fallow party master issue #2 by @SDClowen in https://github.com/SDClowen/RSBot/pull/544
![image](https://user-images.githubusercontent.com/14323838/199607711-f243ea31-0070-4a2e-b1d1-6c00bfc3966f.png)
* Add teleport skill feature by @ngoedde in https://github.com/SDClowen/RSBot/pull/546
![image](https://user-images.githubusercontent.com/12081190/200035124-40f3e89a-48a3-4f7a-b98f-bf5e22a8e742.png)
* Fix statistics kill counter by @ngoedde in https://github.com/SDClowen/RSBot/pull/545
* Fix teleportation position packet by @ngoedde in https://github.com/SDClowen/RSBot/pull/547
* Add command center plugin by @ngoedde in https://github.com/SDClowen/RSBot/pull/551
![image](https://user-images.githubusercontent.com/12081190/200313766-dc40043f-ed10-4b41-8441-71256f971bfa.png)
![image](https://user-images.githubusercontent.com/12081190/200313608-624aa270-12fe-4151-aa9b-980e9841824b.png)
![image](https://user-images.githubusercontent.com/12081190/200313651-5ec3426f-c41e-4bc7-a940-3206d03848b1.png)
![image](https://user-images.githubusercontent.com/12081190/200313902-e039527c-5c13-4e32-9a40-ea4633dba1fb.png)

* Fix game ready state and transport position after spawn by @ngoedde in https://github.com/SDClowen/RSBot/pull/555
* Inventory Plugin: Feature Auto Use Item According To Purpose by @SDClowen in https://github.com/SDClowen/RSBot/pull/559
![image](https://user-images.githubusercontent.com/14323838/201479050-36ca5e92-d3ef-4a9a-9a9f-63dbdd45f077.png)
* Add lure bot by @ngoedde in https://github.com/SDClowen/RSBot/pull/564
![image](https://user-images.githubusercontent.com/12081190/201496390-b08e9131-d30d-4af2-9ea0-c3af88784eb9.png)
![image](https://user-images.githubusercontent.com/12081190/201496377-58106d0a-8035-4900-ad58-499e7925f5ff.png)
* Feature: RSBot Setup will be setting up .NET 6.0 desktop runtime automatically in the case if it's not exists on the target computer. 
**Full Changelog**: https://github.com/SDClowen/RSBot/compare/v2.5.1...v2.6

<hr /><em>This discussion was created from the release <a href='https://github.com/SDClowen/RSBot/releases/tag/v2.6'>RSBot v2.6</a>.</em>

**Comment by AnhVanhihi** [comment]
I got this error please help me fix it
![image](https://user-images.githubusercontent.com/111055373/232851223-778c5296-8ff6-4cd4-970c-d733efff2dff.png)

**Comment by AnhVanhihi** [comment]
I need to turn on the lure
